spec/orm/activerecord_spec.rb

```diff
@@ -1195,6 +1197,14 @@ def monkey
     end
 
     describe "#remote_images_urls=" do
+      before do
+        sham_rack_app = ShamRack.at('www.example.com').stub
+        sham_rack_app.register_resource('/test.jpg', File.read(file_path('test.jpg')), 'images/jpg')
```

Comment:
Yeah, a PR would be great. Much easier for me to look over on my mobile (out of town at the moment).